src/CoreWCF.Http/tests/HttpSysTests.cs

```diff
@@ -24,8 +24,7 @@ public HttpSysTests(ITestOutputHelper output)
             _output = output;
         }
 
-        [Fact]
-        [Trait("Category", "WindowsOnly")]
+        [WindowsNetCoreOnlyFact]
 #if NET5_0_OR_GREATER
```

Comment:
I believe this one should be WindowsOnly. The #if around NET5_0 was for the benefit of the SupportedOSPlatform attribute.. The UseHttpSys api has this attribute, which then means it's needed to be added to CreateHttpSysBuilder. Without these attributes, you get compiler warnings as the test builds target net6.0 and not net6.0-windows. These attributes aren't available for .NET Framework which is why there's the conditional compilation around the attribute. The test runs just fine on .NET Framework though.

Reply:
done, thank you for the explanation.